File: Prueba_final/Controladores/LoginStatus.cs
using Prueba_final.Clases;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Prueba_final.Controladores
{
    public class LoginStatus
    {
        private static Usuario usuario = null;
        public static void InsertarUsuario(Usuario u) => usuario = u;

        public static Usuario GetUsuario() => usuario;
        public static bool Logeado()
        {
            bool logeado = false;
            if (usuario != null)
            {
                logeado = true;
            }
            return logeado;
        }
    }
}

File: Programacion de servicios y procesos (PSPRO)/Segundo bloque/Android/Ejercicios/preguntas/app/src/main/java/com/example/preguntas/Controller/LoginStatus.java
package com.example.preguntas.Controller;

import com.example.preguntas.Clases.Usuario;

public class LoginStatus {
    private static Usuario usuario;

    public static void login(Usuario u){
        usuario = u;
    }
    public static Usuario devolverUsuario(){
        return usuario;
    }
    public static boolean logeado(){
        boolean logeado = false;
        if (usuario != null)
        {
            logeado = true;
        }
        return logeado;
    }
    public static void desconectar(){
        usuario = null;
    }
}
